src/util/system.cpp

```diff
@@ -1200,25 +1200,30 @@ UniValue RunCommandParseJSON(const std::string& str_command, const std::string&
 
     if (str_command.empty()) return UniValue::VNULL;
 
-    bp::child c(
+    // This is here because boost::process quotes the argument given to
+    // system when used in combination with boost::process::shell.
+    // https://github.com/boostorg/process/issues/95
+    auto exit =
+#ifdef _WIN32
+    bp::system(
+    bp::shell(), cmd, bp::std_out > stdout_stream);
+#else
+    bp::system(bp::shell(),
+        "-c",
         str_command,
         bp::std_out > stdout_stream,
         bp::std_err > stderr_stream,
         bp::std_in < stdin_stream
     );
-    if (!str_std_in.empty()) {
-        stdin_stream << str_std_in << std::endl;
-    }
+#endif
+    stdin_stream << str_std_in << std::endl;
```

Comment:
This shouldn't work: bp::system won't return until the child process is complete, which means it's done with stdin...